lib/index.test.js

```diff
@@ -1,16 +1,16 @@
 const path = require('path');
-const CLIEngine = require('eslint').CLIEngine;
+const { ESLint } = require('eslint');
 
 describe('config', () => {
-    it('works', () => {
-        const cli = new CLIEngine({
+    it('works', async () => {
+        const eslint = new ESLint({
             useEslintrc: false,
-            configFile: path.join(__dirname, 'index.js'),
+            overrideConfigFile: path.join(__dirname, 'index.js'),
         });
 
-        expect(cli.executeOnText('var foo\n')).toMatchObject({
-            errorCount: 3,
-            warningCount: 0,
-        });
+        const results = await eslint.lintText('var foo\n');
+
+        expect(results[0].errorCount).toEqual(3);
+        expect(results[0].warningCount).toEqual(0);
```

Comment:
What is the point of this test? I'm not getting it.

Reply:
This just runs the linter on the `var foo\n` string and brings up errors associated with it. It's like a double lock of the config, so if someone changes the config they'd have to update this test too.